scripts/build.sh

```diff
@@ -17,3 +17,7 @@ rm -f build/conf/bblayers.conf || die "failed to nuke bblayers.conf"
 rm -f build/conf/local.conf || die "failed to nuke local.conf"
 
 ./scripts/containerize.sh bitbake ${BUILD_TARGETS} || die "failed to build"
+
+echo "DEBUG_BUILD=1" >> build/conf/local.conf
```

Comment:
This needs to use bitbake syntax, i.e. DEBUG_BUILD="1", so:

echo 'DEBUG_BUILD="1"' >> build/conf/local.conf